backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/keyboardheight/StandardKeyboardHeightProvider.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * This file is part of Siebe Projects samples.
+ *
+ * Siebe Projects samples is free software: you can redistribute it and/or modify
+ * it under the terms of the Lesser GNU General Public License as published by
```

Comment:
@intrigus This block is more concerning, though it's still just from a sample, and LGPL (v3) is more compatible than GPL (v2, which OpenJDK uses) with permissive licenses. I'm not sure if LGPL v3 is compatible enough with Apache v2; I think it is, though I personally steer clear of any GNU-licensed code in my non-GNU-licensed projects.

Reply:
The "Siebe Projects samples" mentioned is here: https://github.com/siebeprojects/samples-keyboardheight

It doesn't seem trivial to reimplement from scratch.

Reply:
https://github.com/siebeprojects/samples-keyboardheight/issues/10

I don't understand this timeline, maybe we can ask him to finish license change?